### ISSUE TITLE
AuthorizationInterface already exists in context object

### ISSUE THREAD

**Issue by sammarcus** [Agence-DnD/PIMGento-2#35]
Hi,

During code compilation after installation I am receiving the following error:

```
	Pimgento\Import\Block\Adminhtml\Import
		Incorrect dependency in class Pimgento\Import\Block\Adminhtml\Import in /opt/bitnami/apps/magento/htdocs/vendor/agencednd/module-pimgento/Import/Block/Adminhtml/Import.php
\Magento\Framework\AuthorizationInterface already exists in context object
```

I had installed this awhile ago and don't recall observing this same issue. Any ideas how to fix?

![image](https://cloud.githubusercontent.com/assets/8689264/15870825/aa4ada0c-2cbf-11e6-87e9-c96a1380cfb9.png)

**Comment by sammarcus** [comment]
Any ideas?

**Comment by magentix** [comment]
Hi,

Sorry for late reply. We fix this on Monday June 20, 2016.

**Comment by magentix** [comment]
Hi,

This has been fixed in release 100.0.11

**Comment by magentix** [comment]
@sammarcus 

Is it ok for you?
